thresher_backend/settings.py

```diff
@@ -66,11 +66,27 @@
 
 ROOT_URLCONF = 'thresher_backend.urls'
 
+TEMPLATES = [
```

Comment:
The old TEMPLATE_* settings were deprecated in 1.8. Django 1.8 generalized using and configuring multiple template engines. We actually aren't using the template engines at the moment since it is all DRF, but the command I use to generate the entity-relationship diagram broke without the new TEMPLATES configuration.
https://docs.djangoproject.com/en/1.10/ref/templates/upgrading/

btw, I pulled the TEMPLATES settings from the settings.py created by `django startproject` using Django 1.10, so they should be solid defaults.

Here are the links for other changes as well:
django.conf.urls.patterns was also [deprecated in 1.8](https://docs.djangoproject.com/en/1.8/ref/urls/#patterns).

The MIDDLEWARE_CLASSES setting changed to MIDDLEWARE in 1.10. 
https://docs.djangoproject.com/en/1.10/topics/http/middleware/

I guess I should double check the CORS middleware `corsheaders.middleware.CorsMiddleware` to [verify compatibility](https://docs.djangoproject.com/en/1.10/topics/http/middleware/#upgrading-middleware).